xa/nodejs/department/dbconfig.ts

```diff
@@ -21,5 +21,7 @@ CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFT
 export default {
     user          : process.env.DEPARTMENTDATASOURCE_USER,
     password      : process.env.DEPARTMENTDATASOURCE_PASSWORD,
-    connectString : process.env.DEPARTMENTDATASOURCE_URL
+    connectString : process.env.DEPARTMENTDATASOURCE_URL,
+    walletLocation: process.env.DEPARTMENTDATASOURCE_WALLETLOCATION,
+    walletPassword: process.env.DEPARTMENTDATASOURCE_WALLETPASSWORD
 };
```

Comment:
Please add a example connection string here for the reference
/*
 Sample connection string:
 connectString : ...
*/